tests/indent/lua/comment.lua

```diff
@@ -5,3 +5,9 @@
   another
   comment
 --]]
+
+  --[[
+    another
+    comment
+    a bit more indented
+  --]]
```

Comment:
I don't think comment should start from arbritary indentation level. Maybe wrap it in a function if that was the intention?

```suggestion
function comment_inside_func()
  --[[
    another
    comment
    a bit more indented
  --]]
end
```

Reply:
we need some mechanism to keep indent relative to beginning of node.